src/Opdex.Platform.Application/EntryHandlers/Blocks/CreateRewindToBlockCommandHandler.cs

```diff
@@ -46,23 +46,24 @@ public async Task<bool> Handle(CreateRewindToBlockCommand request, CancellationT
                 throw new ArgumentOutOfRangeException(nameof(request.Block), "Rewind request exceeds maximum rewind limit.");
             }
 
-            var rewound = await _mediator.Send(new MakeRewindToBlockCommand(request.Block));
+            using var cancellationTokenSource = new CancellationTokenSource(TimeSpan.FromMinutes(1));
```

Comment:
I don't know if this really benefits us setting a minute long token. It might cause more problems then it fixes since a cancellation here would cause bad data issues.

Reply:
If it gets cancelled I believe it'll rollback, just like it would if it were to fail. The idea is to prevent it hanging. Happy to remove it though, as the indexer should come around and force unlock itself with an alert, so it'll be evident something is wrong.

Reply:
I think we should pull this out for now.

- Wed have to adjust MakeRewindToBlockComand/SelectRewindToBlockCommand to pass the cancellation token all the way down
- We'd have to do the same for all of the other commands below this comment and all children they call and so on, thats a big chain.
- Then we could still run into a situation if a valid rewind does take more than a minute, stored procedure deletes back to block, then have half of the below commands actually finish processing, leaving us in a bad state requiring another rewind.